package/MDAnalysis/coordinates/GRO.py

```diff
@@ -97,47 +94,28 @@
 from ..lib import util
 
 
-class Timestep(base.Timestep):
-    _ts_order_x = [0, 3, 4]
-    _ts_order_y = [5, 1, 6]
-    _ts_order_z = [7, 8, 2]
-
-    def _init_unitcell(self):
-        return np.zeros(9, dtype=np.float32)
+"""unitcell dimensions (A, B, C, alpha, beta, gamma)
```

Comment:
Should these docs be properly formatted for module level docs? From the diff I cannot really see what's going on.

Reply:
yeah this is an old comment from withing the format specific timestep, I kept it around as it seemd useful for developers